frontend/tests/.gitignore

```diff
@@ -0,0 +1 @@
+/elm-stuff/
```

Comment:
This file can be deleted since `.gitignore` in root directory already deals with `elm-stuff`.